test/testvalueflow.cpp

```diff
@@ -3126,7 +3126,7 @@ class TestValueFlow : public TestFixture {
         ASSERT_EQUALS(true, values.size()==1U && values.front().isUninitValue());
 
         code = "void f() {\n"
-               "    C *c;\n"
+               "    const C *c;\n"
```

Comment:
Would it be helpful to keep a regression test for the previous example?

Reply:
What do you mean? The previous example won't pass. 

Although we should probably fix it because even calling `c->x()` with a mutable method it would still be using uninitialized variable. Maybe we need to add a flag to `isVariableModified` to skip checking member functions.

Reply:
I meant keep the new result of the previous as a regression test.

Reply:
I see, but I think the new result is wrong. I could add it as a `TODO_ASSERT`.

Reply:
yes I would like a `TODO_ASSERT`. `c` is uninitialized no matter if it is const or not.